frontend/src/component/project/Project/ProjectHealth/ReportTable/ReportCard/ReportCard.tsx

```diff
@@ -95,9 +100,29 @@ export const ReportCard = ({ healthReport }: IReportCardProps) => {
             <span>
                 {healthReport.potentiallyStaleCount} potentially stale toggles
             </span>
+
         </StyledBoxStale>
     );
 
+    const renderStalenessInfoIcon = () => (
+        <HtmlTooltip title={
+            <>
+                If your toggle exceeds the expected lifetime of it's toggle type it will be marked as potentially stale.
+                <br/>
```

Comment:
again should be styling in either a or add a box around the sentence above and style it